joint4_controller:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: joint4
  joint_speed: 1.0
  motor:
    id: 5
    init: 0
    min: 0
    max: 1023